sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/src/AzureKeyVaultConfigurationOptions.cs

```diff
@@ -11,7 +11,7 @@ namespace Azure.Extensions.AspNetCore.Configuration.Secrets
     /// <summary>
     /// Options class used by the <see cref="AzureKeyVaultConfigurationExtensions"/>.
     /// </summary>
-    internal class AzureKeyVaultConfigurationOptions
+    public class AzureKeyVaultConfigurationOptions
```

Comment:
Let's only keep the `Manager` and `ReloadInterval` properties publicly on this type and remove the client and parametrized ctor.

Reply:
I believe I've changed the class to match this request.